src/types/txpool.rs

```diff
@@ -0,0 +1,204 @@
+use std::collections::HashMap;
+use crate::types::{Transaction};
+use serde::{Deserialize, Serialize};
+use crate::types::U64;
+
+/// Transaction Pool Content Info
+#[derive(Debug, Default, Clone, PartialEq, Deserialize, Serialize)]
+pub struct TxpoolContentInfo {
+    /// pending tx
+    pub pending: HashMap<String, HashMap<String, Transaction>>,
+    /// queued tx
+    pub queued: HashMap<String, HashMap<String, Transaction>>,
+}
+
+/// Transaction Pool Inspect Info
```

Comment:
Can we expand on the docs or at least point to documentation of these structs? I'd be more interested in the `String` format of the transaction inspection info - i.e. what things can be found there and can we actually parse it?